lc-0226-invert-binary-tree/src/test/java/solution/CustomCsvArgTest.java

```diff
@@ -0,0 +1,184 @@
+package solution;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
+
+public class CustomCsvArgTest {
```

Comment:
I'm grouping generating binary tree List<Integer> from String and TreeNode in one static class. This works and I like keeping these together, but lmk if there's a better way to do this.